process/process.lua

```diff
@@ -356,7 +356,10 @@ function process.handle(msg, _)
     function (msg)
       return msg.Action == "Eval" and Owner == msg.From
     end,
-    require('.eval')(ao)
+    function (msg)    
+      print('{ "_e": 1, "Message-Id": "' .. msg.Id .. '", "From": "' .. msg.From .. '", "Action": "' .. msg.Action .. '", "Timestamp": ' .. tonumber(msg.Timestamp) .. '}')
+      return require('.eval')(ao)(msg)
```

Comment:
Why is the `return` necessary here? The function returned by `require('.eval')(ao)` itself does not return anything in any of its exit cases. So it stands to reason that we also do not need to return anything in our wrapped handler of it.

Reply:
a3f3523